src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -92,17 +94,28 @@ class FilterConsensusReads
   @arg(flag='o', doc="Output SAM or BAM file.") val output: PathToBam,
   @arg(flag='r', doc="Reference fasta file.") val ref: PathToFasta,
   @arg(flag='R', doc="Reverse [complement] per base tags on reverse strand reads.") val reversePerBaseTags: Boolean = false,
-  @arg(flag='M', minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
+
+ // Filters that are applied both per-base and per-read
+  @arg(flag='M', group="Per-read/Per-base", minElements=1, maxElements=3, doc="The minimum number of reads supporting a consensus base/read.")
   val minReads: Seq[Int],
-  @arg(flag='E', minElements=1, maxElements=3, doc="The maximum raw-read error rate across the entire consensus read.")
+
+ // Filters that are applied per-read
+  @arg(flag='E', group="Per-read", minElements=1, maxElements=3, doc="The maximum raw-read error rate across the entire consensus read.")
   val maxReadErrorRate: Seq[Double] = Seq(0.025),
-  @arg(flag='e', minElements=1, maxElements=3, doc="The maximum error rate for a single consensus base.")
+  @arg(flag='n', group="Per-read", doc="Maximum fraction of no-calls in the read after filtering.")
+  val maxNoCallFraction: Double = 0.2,
+  @arg(flag='m', group="Per-read", minElements=0, maxElements=3, doc="The minimum number of reads supporting a consensus base at any base in the consensus read")
+  val minDepth: Seq[Int] = Seq.empty,
```

Comment:
I find this _super_ confusing.  To have two options `min-reads` and `min-depth` that do subtly but significantly different things is just going to cause errors.  I think that it will also have un-desirable results when reads have been quality-trimmed prior to consensus calling, leading to a lot of reads having lots of bases above `min-reads`, but some below `min-depth` at the end.